variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "prefix" {
+ description = "The prefix used for all resources in this environment"
+ type = string
+ default = "terraformed"
+}
+
+variable "location" {
+ description = "The Azure location where all resources in this deployment should be created"
+ default = "uksouth"
+}
+
+variable "ANON" {
+  description = "ReadOnly Test GIT Account"
+  type        = string
+  default     = "TheLegendaryPanTester"
+}
+
+variable "ANON_USER" {
+  description = "ReadWrite GIT Account"
+  type        = string
+  default     = "TheLegendaryPan"
+}
+
+variable "flask_app" {
+  description = "The FLASK_APP environment variable"
+  type        = string
+  default     = "app"
+}
+
+variable "flask_env" {
+  description = "The FLASK_ENV environment variable"
+  type        = string
+  default     = "development"
+}
+
+variable "SECRET_KEY" {
+  description = "The FLASK_ENV environment variable"
+  type        = string
+  default     = "secret-key"
```

Comment:
Although we don't store anything particularly sensitive in our sessions, it's still a bad habit to publicise this secret value